apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Tests/BigQueryNumericTest.cs

```diff
@@ -219,6 +219,9 @@ public void ToDecimal_Overflow(LossOfPrecisionHandling handling, bool negate)
         [InlineData("+.5")]
         [InlineData("1e6")]
         [InlineData("1e-6")]
+        // Other culture formatting
```

Comment:
It might be worth doing this in a separate test which sets the thread culture to a culture where this *would* be valid first.

Reply:
Agreed, I will merge this one since it's LGTMed already and submit a new one with the new test you are describing.